### PR TITLE
Fields validation errors for unauthorized users

### PR DESCRIPTION
**Pull request by Ivan-Zhdanovich**

### CODE REVIEW

**Comment by AlexMakovetskiy** [comment]
const validateRepeatPassword = (value: string | undefined) => {
		const password = watch(InputType.Password);
		return value === password || errorPasswordRepeat;
	};
	
	can you update validateRepeatPassword function in src/components/mainLayout/signUpForm/signUpForm.tsx with this code?

File: src/utils/formHelpers.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ class FormHelpers {
 		} else if (password.length > minPasswordLength && !/(?=.*\d)/.test(password)) {
 			messages.push(errorPasswordNumber);
 		} else if (!passwordPattern.test(password)) {
-			messages.push(errorPasswordRules);
+			messages.push(errorPasswordIncorrect);
```
Comment (AlexMakovetskiy) [inline src/utils/formHelpers.ts:40]: better to leave the previous value

**Comment by AlexMakovetskiy** [comment]
constructPasswordMessage = (password: string) => {
		const messages = [];
		const minPasswordLength = 6;
		if (password.length < minPasswordLength) {
			messages.push(errorPasswordLength);
		} else if (password.length >= minPasswordLength && !/(?=.*\d)/.test(password)) {
			messages.push(errorPasswordNumber);
		} else if (!passwordPattern.test(password)) {
			messages.push(errorPasswordRules);
		}
		if (messages.length === 0) {
			return null;
		}
		return messages.join(" ");
	};
	can you update constructPasswordMessage method in src/utils/formHelpers.ts with this code?

**Comment by AlexMakovetskiy** [comment]
} else if (password.length >= minPasswordLength && !/(?=.*\d)/.test(password)) {
		
		can you update 38 line in constructPasswordMessage method in src/utils/formHelpers.ts with this code?

**Comment by Ivan-Zhdanovich** [comment]
> getPasswordError

done

**Comment by Ivan-Zhdanovich** [comment]
> ```
> 	} else if (password.length >= minPasswordLength && !/(?=.*\d)/.test(password)) {
> 	
> 	can you update 38 line in constructPasswordMessage method in src/utils/formHelpers.ts with this code?
> ```

done

**Comment by Ivan-Zhdanovich** [comment]
> ```
> 	} else if (password.length >= minPasswordLength && !/(?=.*\d)/.test(password)) {
> 	
> 	can you update 38 line in constructPasswordMessage method in src/utils/formHelpers.ts with this code?
> ```

done

**Comment by Ivan-Zhdanovich** [comment]
> ```
> 	} else if (password.length >= minPasswordLength && !/(?=.*\d)/.test(password)) {
> 	
> 	can you update 38 line in constructPasswordMessage method in src/utils/formHelpers.ts with this code?
> ```

done

**Comment by Ivan-Zhdanovich** [comment]
> ```
> const validateRepeatPassword = (value: string | undefined) => {
> 	const password = watch(InputType.Password);
> 	return value === password || errorPasswordRepeat;
> };
> 
> can you update validateRepeatPassword function in src/components/mainLayout/signUpForm/signUpForm.tsx with this code?
> ```

done

**Review by AlexMakovetskiy — APPROVED** [review]

**Pull request merged**